### ISSUE TITLE
Seplos V 3.0 "SP6*"  (BMS B-48100 / B-48200)

### ISSUE THREAD

**Issue by wrobela** [patman15/BMS_BLE-HA#130]
### Checklist

- [X] I have filled out the template to the best of my ability.
- [X] I have understood that this integration is only about battery management systems with Bluetooth Low Energy support.
- [X] This only contains 1 feature request (if you have multiple feature requests, open one feature request for each feature request).
- [X] This issue is not a duplicate of any [previous feature requests](https://github.com/patman15/BMS_BLE-HA/issues?q=is%3Aissue+label%3A%22Feature+Request%22+).

### Please describe the enhancement you would like to have.

Is it possible to add support for Seplos BMS V3.0 with "SP6*"  (e.g. B-48100 / B-48200) ? Unfortunately I am not an expert in this field and I cannot do it myself. I see that BMS from SP0* to SP2* are supported. I would be very grateful for your help.

### Additional context

[14:27:44][V][esp32_ble:348]: (BLE) gap_event_handler - 3
[14:27:44][V][bluetooth_proxy:058]: Proxying raw packet from C0:D6:3C:4C:D8:94, length 31. RSSI: -81 dB
[14:27:44][V][bluetooth_proxy:061]: Proxying 1 packets
[14:27:44][VV][api.service:345]: send_bluetooth_le_raw_advertisements_response: BluetoothLERawAdvertisementsResponse {
  advertisements: BluetoothLERawAdvertisement {
  address: 212026367203476
  rssi: -81
  address_type: 1
  data: '	SP66B2404270002       \xf0\xff'
}
}
[14:27:44][VV][esp32_ble_tracker:434]: Parse Result:
[14:27:44][VV][esp32_ble_tracker:453]:   Address: C0:D6:3C:4C:D8:94 (RANDOM)
[14:27:44][VV][esp32_ble_tracker:456]:   RSSI: -81
[14:27:44][VV][esp32_ble_tracker:457]:   Name: 'SP66B2404270002       '
[14:27:44][VV][esp32_ble_tracker:465]:   Ad Flag: 6
[14:27:44][VV][esp32_ble_tracker:468]:   Service UUID: 0xFFF0
[14:27:44][VV][esp32_ble_tracker:489]:   Adv data: 02.01.06.17.09.53.50.36.36.42.32.34.30.34.32.37.30.30.30.32.20.20.20.20.20.20.20.03.03.F0.FF (31)

[14:27:45][V][bluetooth_proxy:058]: Proxying raw packet from C0:D6:3C:4C:D8:94, length 55. RSSI: -84 dB
[14:27:45][V][bluetooth_proxy:061]: Proxying 1 packets
[14:27:45][VV][api.service:345]: send_bluetooth_le_raw_advertisements_response: BluetoothLERawAdvertisementsResponse {
  advertisements: BluetoothLERawAdvertisement {
  address: 212026367203476
  rssi: -84
  address_type: 1
  data: '	SP66B2404270002       \xf0\xff	SP66B2404270002       '
}
}
[14:27:45][VV][esp32_ble_tracker:434]: Parse Result:
[14:27:45][VV][esp32_ble_tracker:453]:   Address: C0:D6:3C:4C:D8:94 (RANDOM)
[14:27:45][VV][esp32_ble_tracker:456]:   RSSI: -84
[14:27:45][VV][esp32_ble_tracker:457]:   Name: 'SP66B2404270002       '
[14:27:45][VV][esp32_ble_tracker:465]:   Ad Flag: 6
[14:27:45][VV][esp32_ble_tracker:468]:   Service UUID: 0xFFF0
[14:27:45][VV][esp32_ble_tracker:489]:   Adv data: 02.01.06.17.09.53.50.36.36.42.32.34.30.34.32.37.30.30.30.32.20.20.20.20.20.20.20.03.03.F0.FF.17.09.53.50.36.36.42.32.34.30.34.32.37.30.30.30.32.20.20.20.20.20.20.20 (55)

**Comment by patman15** [comment]
Should be an easy fix. Can you try the version from the attached branch?

**Comment by wrobela** [comment]
Thank you! It works, it was a really quick fix, BMS has been detected. 
![BMS_Seplos](https://github.com/user-attachments/assets/6386f736-ec03-4588-8e32-cd327c3e70cf)

Unfortunately, this leads me to another question. Since there are two BMS connected in parallel, how do I show information from both devices? The phone app allows you to switch between data from both devices. Is there also a way to get data from both devices?

**Comment by patman15** [comment]
> Is there also a way to get data from both devices?

The data should be the combined data from both devices. If you are looking for the cell voltages, they are as attribute to the `delta voltage`sensor, see the [description](https://github.com/patman15/BMS_BLE-HA#provided-information). You should see all cells from both packs. If not, please provide a debug log, so I can check what the issue could be.

**Comment by wrobela** [comment]
Once again, thank you very much for your quick help. Now everything works correctly.
![ble_seplos_v30](https://github.com/user-attachments/assets/75d293fe-8b61-4568-a4be-473f06892edf)
